scripts/notifications.coffee

```diff
@@ -0,0 +1,78 @@
+# Description:
+#   Notifications for botgt
+# Commands:
+#   hubot notify <message> - notify everyone with something
+#   hubot notify #channel1 ... #channeln !medium1 ... !mediumn <message>  - notify certain channels with something channels and mediums are optional but order matters (do not mix channels and mediums)
+# Author:
+#   Jacob Zipper
+
+request = require "request-promise"
+adminKey = (Buffer.from process.env.BUZZER_ADMIN_KEY_SECRET).toString 'base64'
+options = {
+  uri: 'https://buzzer.hack.gt/graphql',
+  qs: {
+    query: '',
+    variables: ''
+  },
+  headers: {
+    'Authorization': 'Basic ' + adminKey
+  },
+  json: true
+}
+
+module.exports = (robot) ->
+  robot.respond /notify (.*)/i, (res) ->
```

Comment:
If the user gives a malformed request, can we print a usage statement instead. EX:
ehsan: @botgt notify
botgt: Error, you didn't supply a message. Usage: `@bogt notify <channel> <message>`